<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../civiltools.qrc">
    <normaloff>:/civiltools/images/automatic.svg</normaloff>:/civiltools/images/automatic.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="1">
           <widget class="QLabel" name="label_5">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>نوع خاک</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="Gui::PrefComboBox" name="importance_factor">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <property name="prefEntry" stdset="0">
             <cstring>importance_factor</cstring>
            </property>
            <property name="prefPath" stdset="0">
             <cstring>Mod/civilTools</cstring>
            </property>
            <item>
             <property name="text">
              <string>0.8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1.0</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1.2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1.4</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="Gui::PrefComboBox" name="soil_type">
            <property name="currentIndex">
             <number>2</number>
            </property>
            <property name="prefEntry" stdset="0">
             <cstring>soiltype</cstring>
            </property>
            <property name="prefPath" stdset="0">
             <cstring>Mod/civilTools</cstring>
            </property>
            <item>
             <property name="text">
              <string>I</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>II</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>III</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>IV</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="label_6">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>ضریب اهمیت</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_3">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>استان</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="Gui::PrefComboBox" name="ostan">
            <property name="prefEntry" stdset="0">
             <cstring>ostan</cstring>
            </property>
            <property name="prefPath" stdset="0">
             <cstring>Mod/civilTools</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_14">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>خطر نسبی</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0" colspan="2">
           <widget class="Gui::PrefCheckBox" name="infill">
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string>اثر میانقاب</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <property name="prefEntry" stdset="0">
             <cstring>is_infill</cstring>
            </property>
            <property name="prefPath" stdset="0">
             <cstring>Mod/Civil</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="Gui::PrefComboBox" name="city">
            <property name="prefEntry" stdset="0">
             <cstring>city</cstring>
            </property>
            <property name="prefPath" stdset="0">
             <cstring>Mod/civilTools</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_4">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>شهر</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Stories For Apply Earthquakes</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="Gui::PrefComboBox" name="bot_x_combo">
               <property name="prefEntry" stdset="0">
                <cstring>ostan</cstring>
               </property>
               <property name="prefPath" stdset="0">
                <cstring>Mod/civilTools</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_12">
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
                <string>پایینترین طبقه</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="Gui::PrefComboBox" name="top_x_combo">
               <property name="prefEntry" stdset="0">
                <cstring>ostan</cstring>
               </property>
               <property name="prefPath" stdset="0">
                <cstring>Mod/civilTools</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="label_13">
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
                <string>بالاترین طبقه</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="7" column="0" colspan="2">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Stories For Height</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="Gui::PrefComboBox" name="top_story_for_height">
               <property name="prefEntry" stdset="0">
                <cstring>ostan</cstring>
               </property>
               <property name="prefPath" stdset="0">
                <cstring>Mod/civilTools</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QCheckBox" name="top_story_for_height_checkbox">
               <property name="layoutDirection">
                <enum>Qt::RightToLeft</enum>
               </property>
               <property name="text">
                <string>طبقه بالای ارتفاعی</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="Gui::PrefDoubleSpinBox" name="height_x">
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="suffix">
                <string> m</string>
               </property>
               <property name="maximum">
                <double>300.000000000000000</double>
               </property>
               <property name="value">
                <double>10.000000000000000</double>
               </property>
               <property name="prefEntry" stdset="0">
                <cstring>building_height</cstring>
               </property>
               <property name="prefPath" stdset="0">
                <cstring>Mod/civilTools</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="label_7">
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
                <string>ارتفاع</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QSpinBox" name="no_of_story_x">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="value">
                <number>3</number>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="label_8">
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
                <string>تعداد طبقات</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QComboBox" name="risk_level">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>خیلی زیاد</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>زیاد</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>متوسط</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>کم</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="cancel_pushbutton">
            <property name="text">
             <string>Cancel</string>
            </property>
            <property name="icon">
             <iconset resource="../civiltools.qrc">
              <normaloff>:/civiltools/images/cancel.svg</normaloff>:/civiltools/images/cancel.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="save_pushbutton">
            <property name="text">
             <string>Save</string>
            </property>
            <property name="icon">
             <iconset resource="../civiltools.qrc">
              <normaloff>:/civiltools/images/save.png</normaloff>:/civiltools/images/save.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>First System</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>X System</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTreeView" name="x_treeview">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Y System</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTreeView" name="y_treeview">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Second System</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="activate_second_system">
           <property name="text">
            <string>Activate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="x_system_label">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>X System</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTreeView" name="x_treeview_2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="y_system_label">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Y System</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTreeView" name="y_treeview_2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Gui::PrefCheckBox" name="show_at_startup">
     <property name="text">
      <string>Show at Startup</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="prefEntry" stdset="0">
      <cstring>show_at_startup</cstring>
     </property>
     <property name="prefPath" stdset="0">
      <cstring>Mod/Civil</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::PrefCheckBox</class>
   <extends>QCheckBox</extends>
   <header>Gui/PrefWidgets.h</header>
  </customwidget>
  <customwidget>
   <class>Gui::PrefComboBox</class>
   <extends>QComboBox</extends>
   <header>Gui/PrefWidgets.h</header>
  </customwidget>
  <customwidget>
   <class>Gui::PrefDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>Gui/PrefWidgets.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../civiltools.qrc"/>
 </resources>
 <connections/>
</ui>
